### ISSUE TITLE
*moved here* HTTP returns 302 for Fatal exception (28)

### ISSUE THREAD

**Issue by b18062a** [HomeACcessoryKid/life-cycle-manager#24]
moved from: https://github.com/maximkulkin/esp-homekit/issues/132#issuecomment-610572015'

**Latest update to continue from:**

I had a read of your main page. I saw the 'how to use it' section, but am clueless how to apply it to https://www.studiopieters.nl/esp8266-homekit-light-switch/. In any case, I just ran git clone https://github.com/HomeACcessoryKid/life-cycle-manager.git, then I checked the changelog.md file and it shows information about 1.9.11 so I'd say I'm up-to-date. I then erased and flashed the ESP01.

After booting I still get the wifi config page but after that, same issue (fatal exception 28), over and over.

What would you advise to do at this point? Thank you.

**Comment by HomeACcessoryKid** [comment]
I would like you to present the log file. Does the log say it is running version 1.9.1+ ?
I also updated the readme around the time you read it, maybe you missed the info how to fix in beginning of the readme.

**Comment by b18062a** [comment]
Success:

Here's the details for anyone else getting this.
1. running a GitHub update of your files won't work. My library files were updated and the readme showed as much but the ESP chip still wasn't. I know this because...
2. using the advice above above and @RavenSystem's #[post](https://github.com/HomeACcessoryKid/life-cycle-manager/issues/5), I read the log file (which is simply the output on the serial monitor upon booting) and I saw that the version was old.
3. I didn't realise that StudioPieters OTABoot.bin is the same as that which is on the front page of the Life Cycle Manager (I thought StudioPieter's was customised). So, I replaced StudioPieters OTABoot.bin file provided with the latest on the Life Cycle Manger page.
4. I erased and reflashed the chip and boom, it booted perfectly after entering my details! 

Winner chicken dinner. Thank you @HomeACcessoryKid, and sorry for being a pain.
